### DISCUSSION TITLE
Personalización

### DISCUSSION THREAD

**Discussion by DonPachos** [community/community#132933] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

Como puedo personalizar mi perfil

**Comment by SamuelOUS** [comment, marked as answer]
Hola otra vez!
Ya que estas interesado en personalizar tu perfil, si agregas un README que tenga tu nombre, este sera un repositorio especial que te permitira meter codigo html que aparecera al inicio en tu perfil, con eso puedes personalizar tu perfil :)
